### PR TITLE
Bricktheworld/develop

### PR DESCRIPTION
**Pull request by SoPat712**
This PR removes unused imports, makes it possible to pin conversations on selection, makes all back buttons on material Icons.Arrow_back instead of icons.arrow_back_ios, and also changes the "You have no chats" text to "You have no archived chats". It also changes the color of the settings icon on Material to be white instead of grey.

### CODE REVIEW

**Comment by zlshames** [comment]
I changed the branch this was merging into. Please use `zach/nightly` from now on since Brandon isn't here anymore.

File: ios/Flutter/Debug.xcconfig [hunk 1]
```diff
@@ -1 +1,2 @@
+#include "Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig"
```
Comment (zlshames) [inline ios/Flutter/Debug.xcconfig:1]: These changes need to be removed from the PR

File: ios/Flutter/Release.xcconfig [hunk 2]
```diff
@@ -1 +1,2 @@
+#include "Pods/Target Support Files/Pods-Runner/Pods-Runner.release.xcconfig"
```
Comment (zlshames) [inline ios/Flutter/Release.xcconfig:1]: These changes need to be removed from the PR

File: ios/Podfile [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# Uncomment this line to define a global platform for your project
```
Comment (zlshames) [inline ios/Podfile:1]: These changes need to be removed from the PR

File: lib/layouts/conversation_view/new_chat_creator/contact_selector_custom_cupertino_textfield.dart [hunk 4]
```diff
@@ -1066,7 +1066,7 @@ class _CupertinoTextFieldState
           selectionControls:
               widget.selectionEnabled ? cupertinoTextSelectionControls : null,
           onChanged: widget.onChanged,
-          onContentCommited: widget.onContentCommited,
```
Comment (zlshames) [inline lib/layouts/conversation_view/new_chat_creator/contact_selector_custom_cupertino_textfield.dart:1069]: You need to revert these changes in the PR

File: lib/layouts/conversation_view/text_field/blue_bubbles_text_field.dart [hunk 5]
```diff
@@ -367,7 +366,7 @@ class BlueBubblesTextFieldState extends State<BlueBubblesTextField>
                       });
                     }
                   },
-                  onContentCommited: onContentCommit,
```
Comment (zlshames) [inline lib/layouts/conversation_view/text_field/blue_bubbles_text_field.dart:385]: Please revert these changes for the PR

File: lib/layouts/conversation_view/text_field/blue_bubbles_text_field.dart [hunk 6]
```diff
@@ -414,7 +413,7 @@ class BlueBubblesTextFieldState extends State<BlueBubblesTextField>
                             : Colors.white,
                         fontSizeDelta: -0.25,
                       ),
-                  onContentCommited: onContentCommit,
```
Comment (zlshames) [inline lib/layouts/conversation_view/text_field/blue_bubbles_text_field.dart:432]: Please revert these changes for the PR

File: lib/layouts/widgets/CustomCupertinoTextField.dart [hunk 7]
```diff
@@ -1062,7 +1062,7 @@ class _CupertinoTextFieldState extends State<CustomCupertinoTextField>
           selectionControls:
               widget.selectionEnabled ? cupertinoTextSelectionControls : null,
           onChanged: widget.onChanged,
-          onContentCommited: widget.onContentCommited,
```
Comment (zlshames) [inline lib/layouts/widgets/CustomCupertinoTextField.dart:1065]: Please revert these changes for the PR

**Comment by SoPat712** [comment]
alright, so should I rewrite this pr into zach/nightly?

File: packages/extended_text_field/pubspec.lock [hunk 8]
```diff
@@ -7,42 +7,42 @@ packages:
       name: async
```
Comment (zlshames) [inline packages/extended_text_field/pubspec.lock:7]: Please revert these changes for the PR

File: packages/extended_text_field/pubspec.lock [hunk 9]
```diff
@@ -99,56 +99,56 @@ packages:
       name: source_span
```
Comment (zlshames) [inline packages/extended_text_field/pubspec.lock:99]: Please revert these changes for the PR

File: packages/gif_text_field/pubspec.lock [hunk 10]
```diff
@@ -7,49 +7,49 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.5.0-nullsafety"
+    version: "2.5.0-nullsafety.1"
```
Comment (zlshames) [inline packages/gif_text_field/pubspec.lock:10]: Please revert these changes for the PR

**Review by zlshames — CHANGES_REQUESTED** [review]
Please see my comments to improve the PR

**Comment by zlshames** [comment]
> alright, so should I rewrite this pr into zach/nightly?

Nope, I changed it for you. Just address the comments above and you'll be golden

**Comment by SoPat712** [comment]
think I just fixed them with [this commit](https://github.com/BlueBubblesApp/BlueBubbles-Android-App/pull/605/commits/8c8bbd9534aba2b025a220b5b6df7a72bde4168a)

**Review by SoPat712 — COMMENTED** [review]
Fixed?

**Comment by SoPat712** [comment]
IDK how this works sorry for bugging so much.

**Comment by zlshames** [comment]
Err, so you made a commit, but it was just new code. It wasn't fixing the comments i made.

**Comment by zlshames** [comment]
I think you fixed some of the issues, but not all. For instance, the pubspec.lock stuff

**Comment by SoPat712** [comment]
I think I just fixed everything, can you please review it again? I removed all the files that weren't specifically changes I made, I think.

**Review by zlshames — APPROVED** [review]
Close Enough

**Comment by SoPat712** [comment]
Lmao

**Pull request merged**